### DISCUSSION TITLE
Integrazione presa Smart Plug su HA

### DISCUSSION THREAD

**Discussion by Picchios** [indomus/forum#1347] [HUB: Home Assistant]
Buongiorno,
sono nuovo nel mondo Home Assistant, vorrei capire se è possibile integrare (ed in caso come procedere) una presa Smart Plug Amazon (quindi configurata in Alexa) dentro Home Assistant.
Vi ringrazio.

**Comment by snakuzzo** [comment]
No...o meglio...di base non puoi farlo. Solitamente si fa il giro al contrario. Esponi un'entità di HA verso Alexa.
Ad ogni modo qualcuno ha trovato il modo, ma tramite un vero e proprio "accrocco" che è decisamente BRUTTO.
Se ti interessa ti basta cercare su google...

```
configure amazon smart plug home assistant
```

...nei primi risultati troverai la risposta ;)

**Reply by Picchios** [reply]
Grazie! Potrei provare, soprattutto per prendere dimestichezza, ma diciamo che la cosa migliore a questo punto è cambiare la presa con qualcosa di compatibile, senza inventarmi amenità.
Grazie.

**Reply by snakuzzo** [reply]
esatto...e se puoi, prendi qualcosa di integrabile senza passare da cloud esterni ;)

**Reply by snakuzzo** [reply]
Quella di amazon magari potresti usarla a monte del server che eventualmente vuoi riavviare da remoto se si impalla
